import statement for AxiosInstance is missing in definition file

When using rapid.js with TypeScript the compiler alerts about a missing name in the definition file.

Reproduce:
```
tsc index.d.ts
index.d.ts(80,10): error TS2304: Cannot find name 'AxiosInstance'.
```

--- a/index.d.ts
+++ b/index.d.ts
@@ -1,4 +1,4 @@
-import { AxiosRequestConfig, AxiosResponse } from 'axios';
+import { AxiosInstance, AxiosRequestConfig, AxiosResponse } from 'axios';
 
 declare enum RequestType {
     GET = 'get',

